LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -201,17 +203,15 @@ public void HandleRegisterGame(RegisterGameClientRequest request)
                     log.Info(string.Format(Messages.LoginSuccess, this.UserName));
                     RegisterGameClientResponse response = new RegisterGameClientResponse
                     {
-                        // Note: If we send AuthInfo back, it will override the one currently set up on the client
-                        // which will cause issues if the client attempts to reconnect at some point.
-                        AuthInfo = new AuthInfo()
-                        {
-                            AccountId = AccountId,
-                            Handle = playerInfo.Handle
-                        },
-                        SessionInfo = SessionManager.GetSessionInfo(request.AuthInfo.AccountId),
+                        AuthInfo = request.AuthInfo, // Send original, if some data is missing on a new instance the game fails
```

Comment:
Do we need to send password back though?

Reply:
It's not even checked anymore, as far as I undeerstand

Reply:
The problem is that in my tests, if you lost connection, the LobbySessionInfo is reseted so we don't keep the SessionToken and ReconnectionSessionToken, so it sends both as 0, in that case the only way to reauthenticate the player is to receive the username and password bacause it tries to reconnect as the first time

Reply:
Now I wonder when these tokens are even used

I think we can put actual reconnection token into AuthInfo.TicketData